debezium-core/src/main/java/io/debezium/config/CommonConnectorConfig.java

```diff
@@ -446,6 +446,14 @@ public static BinaryHandlingMode parse(String value, String defaultValue) {
             .withImportance(Importance.MEDIUM)
             .withDescription("The name of the data collection that is used to send signals/commands to Debezium. Signaling is disabled when not set.");
 
+    public static final Field TRANSACTION_TOPIC_PREFIX = Field.create("transaction.topic.prefix")
+            .withDisplayName("Transaction topic prefix")
+            .withGroup(Field.createGroupEntry(Field.Group.ADVANCED, 21))
+            .withType(Type.STRING)
+            .withWidth(Width.MEDIUM)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("The prefix to be used for the transaction topic name.");
+
```

Comment:
This needs to be documented in the `documentation/modules/ROOT/pages/connectors/*.adoc` files for connectors that support transaction metadata topics.  See the sections where we discuss connector configurations and add an entry to that table for each connector describing this option.